agentlocal/agent_local.go

```diff
@@ -131,16 +133,27 @@ func (s *Server) Status(ctx context.Context, req *agentlocalpb.StatusRequest) (*
 		connected = false
 		md = new(agentpb.AgentServerMetadata)
 	}
+	var clockDrift *duration.Duration
+	var latency *duration.Duration
+	if req.GetNetworkInfo {
```

Comment:
And if `connected`? Otherwise, it seems like we can return `connected = false` and non-zero drift/latency if we are unlucky enough to connect between `GetAgentServerMetadata` and `GetNetworkInformation` calls.